hadoop-hdds/pom.xml

```diff
@@ -229,6 +229,16 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>picocli</artifactId>
       <version>3.9.6</version>
     </dependency>
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-stub</artifactId>
```

Comment:
What is the requirement of this grpc dependency ? We already have grpc dependency for datanode client proto files.

Reply:
As we have service entries in the proto files, the service stubs are also generated which require these jar files (I got class not found exception without these dependencies).

I tried to use the ratis grpc (with search and replace all the package names similar what we have in the datanode) but it didn't work very well as some common package names (like ServiceException) are also renamed (=package rename). But Hadoop rpc couldn't work with the renamed exceptions.

Finally it turned out the simplest way just to add these dependencies.

And I would like to experiment with grpc for the heartbeat protocol which requires these jar files anyway.

I think it's safe to keep them as of now.